integ/test_init/integ_test/README.md

```diff
@@ -0,0 +1,58 @@
+
+# Welcome to your CDK Python project!
```

Comment:
If this is the README for our integ test, it needs to explain what the integ test does for init. The CDK stuff is great, but we need both.

Reply:
Oh I see that's in the other readme..

Reply:
I think its fine actually, just want to remove the blank sentence